game.py

```diff
@@ -0,0 +1,141 @@
+import random
+import string
+
+
+class Game:
+    def __init__(self, size=3, characters=string.ascii_letters, target=100):
```

Comment:
why do we have characters here, i did not see that being used while creating the Game object

Reply:
True! I changed this, so the characters are declared inside the init method. I did this so they can be used as a default in case no other characters are declared. Or do you think it is better not to? Also, I implemented a new function set_mode(). This function reduces the number of characters to play at a certain level of difficulty.